oidc/pkce.go

```diff
@@ -160,16 +160,21 @@ func getToken(code, codeVerifier string) (pkceResponse PKCEResponse, err error)
 		pkceResponse.ErrorDescription = "Unable to perform POST request to Auth0."
 		return pkceResponse, microerror.Maskf(authorizationError, pkceResponse.Error)
 	}
-
 	defer res.Body.Close()
+
 	body, err := ioutil.ReadAll(res.Body)
 	if err != nil {
 		pkceResponse.Error = "unknown error"
 		pkceResponse.ErrorDescription = "Got an unparseable error from Auth0. Possibly the Auth0 service is down. Try again later."
 		return pkceResponse, microerror.Maskf(authorizationError, pkceResponse.Error)
 	}
 
-	json.Unmarshal(body, &pkceResponse)
+	err = json.Unmarshal(body, &pkceResponse)
+	if err != nil {
+		pkceResponse.Error = "unknown error"
```

Comment:
These error functions / descriptors look weird - not sure if it's something to worry about here.

Reply:
changing that `PKCEResponse` struct would also end in changes in `gsctl`. That should definitely be done in a separate PR.